sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -114,14 +120,16 @@ public AuthResult(
       final FrameworkConfig frameworkConfig,
       final PlannerContext plannerContext,
       final SqlEngine engine,
-      final PlannerHook hook
+      final PlannerHook hook,
+      boolean allowSetStatementsToBuildContext
```

Comment:
`final` please. It's good to be consistent and either use `final` on all fields or none of them.

Reply:
done